### PR TITLE
Remove director limit for SME translation 

### PR DESCRIPTION
**Pull request by thejameshayes**
I mean I think that's what this does.

### CODE REVIEW

File: sme_finance_application_schema/translations.py [hunk 1]
```diff
@@ -27,35 +27,81 @@ def finance_application_v3_to_sme_contact_v3(finance_application):
...
         directors = [x for x in finance_application['actors'] if x['role'] == 'director']
-        if directors:
-            if len(directors) > 1:
-                raise Exception
-            director = directors[0]
+        for director in directors:
             if 'value_of_property_equity' in director:
-                sme_v5['directors_houses'] = director['value_of_property_equity']
+                director_equity = ['value_of_property_equity']
+                if sme_v5.get('directors_houses'):
```
Comment (jmstaley) [inline sme_finance_application_schema/translations.py]: maybe something like:

```
sme_v5.setdefault('directors_houses', 0)
sme_v5['directors_houses'] += director['value_of_property_equity']
```

makes it a little more compact, same for the pattern below in the pension section

File: sme_finance_application_schema/translations.py [hunk 2]
```diff
@@ -27,35 +27,81 @@ def finance_application_v3_to_sme_contact_v3(finance_application):
...
     if finance_application.get('actors'):
         directors = [x for x in finance_application['actors'] if x['role'] == 'director']
-        if directors:
-            if len(directors) > 1:
-                raise Exception
-            director = directors[0]
+        for director in directors:
             if 'value_of_property_equity' in director:
-                sme_v5['directors_houses'] = director['value_of_property_equity']
+                director_equity = ['value_of_property_equity']
```
Comment (jmstaley) [inline sme_finance_application_schema/translations.py]: you missed the dict should be `director['value_of_property_equity']`

File: sme_finance_application_schema/translations.py [hunk 3]
```diff
@@ -27,35 +27,81 @@ def finance_application_v3_to_sme_contact_v3(finance_application):
...
+                    'first_time': 0,
+                    'had_finance_before': 1,
+                    'expert': 2,
+                }
+                reverse_familiarity_table = dict((reversed(item) for item in familiarity_table.items()))
+
+                if sme_v5.get('familiarity_with_financing'):
+                    max_familiarity = max(
+                        familiarity_table[director_familiarity],
+                        familiarity_table[director['familiarity_with_financing']]
```
Comment (jmstaley) [inline sme_finance_application_schema/translations.py]: this doesn't make sense to me, aren't they the same value?

at the top of the if you are setting `director_familiarity = director['familiarity_with_financing']` then using that to access the table and at the same time doing `familiarity_table[director['familiarity_with_financing']]` this is going to result in the same value being passed into max twice, which makes it redundant.

If i'm reading the purpose correctly here wouldn't you be better off doing something like (not exactly):
```python
max_familiarity = max(
  familiarity_table[directory_familiarity],
  sme_v5['familiarity_with_financing']
)
```
I think you got this right in the credit rating section below

File: sme_finance_application_schema/translations.py (hunk 2 above)
Comment (cybojenix) [inline sme_finance_application_schema/translations.py]: ```director_equity = director['value_of_property_equity']```

File: sme_finance_application_schema/translations.py [hunk 4]
```diff
@@ -27,35 +27,81 @@ def finance_application_v3_to_sme_contact_v3(finance_application):
...
+                director_familiarity = director['familiarity_with_financing']
+
+                familiarity_table = {
+                    'first_time': 0,
+                    'had_finance_before': 1,
+                    'expert': 2,
+                }
+                reverse_familiarity_table = dict((reversed(item) for item in familiarity_table.items()))
+
+                if sme_v5.get('familiarity_with_financing'):
```
Comment (cybojenix) [inline sme_finance_application_schema/translations.py]: `max` takes a key to determine order (same as `sorted`), so you can pass in `list().index` to determine orders of strings

File: sme_finance_application_schema/translations.py [hunk 5]
```diff
@@ -27,35 +27,81 @@ def finance_application_v3_to_sme_contact_v3(finance_application):
...
+                    )
+                    sme_v5['familiarity_with_financing'] = reverse_familiarity_table[max_familiarity]
+                else:
+                    sme_v5['familiarity_with_financing'] = director_familiarity
+
             if 'personal_credit_rating' in director:
-                sme_v5['personal_credit_ratings'] = director['personal_credit_rating']
+                director_credit_rating = director['personal_credit_rating']
+
+                credit_table = {
```
Comment (cybojenix) [inline sme_finance_application_schema/translations.py]: ditto re `max` taking key

File: sme_finance_application_schema/translations.py [hunk 6]
```diff
@@ -27,35 +27,74 @@ def finance_application_v3_to_sme_contact_v3(finance_application):
...
+
+                credit_list = [
+                    'very_poor',
+                    'poor',
+                    'ok',
+                    'good',
+                    'excellent',
+                ]
+
+                # Min is probably not the best function here
```
Comment (thejameshayes) [inline sme_finance_application_schema/translations.py]: Who do I need to talk with to sort this out?

**Pull request merged**